static/js/tools/statsvar_menu.tsx

```diff
@@ -3,17 +3,18 @@ import axios from "axios";
 import hierarchy from "../../../tools/pv_tree_generator/hierarchy_top.json";
 
 const jsonPath = "data/hierarchy_statsvar.json";
-export const SEP="'";
+export const SEP = "'";
 
 interface NodePropType {
   l: string; // label
   c: number; // count
-  cd: NodePropType[];  // children
-  t: string;  // type
+  cd: NodePropType[]; // children
+  t: string; // type
   sv: string;
   updateUrl: (statvar: string, add: boolean) => void;
   nodePath: string;
   svPaths: string[][];
+  svAvai: string[];
```

Comment:
maybe rename to svValid? "Avai" is not that easy to guess the meaning

Reply:
done.